Adyen.xcodeproj/project.pbxproj

```diff
@@ -5996,7 +5996,7 @@
 				CODE_SIGN_ENTITLEMENTS = Demo/Shared.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Distribution";
 				CODE_SIGN_STYLE = Manual;
-				CURRENT_PROJECT_VERSION = 9;
+				CURRENT_PROJECT_VERSION = 1;
```

Comment:
Why this thing has changed?

Reply:
its the demo app, since we bumped the version we can reset the build number, right?